.github/pull_request_template.md

```diff
@@ -27,6 +27,13 @@ https://github.com/DataBiosphere/azul/issues/NUMBER
 - [ ] Added announcement to PR description                  <sub>or this PR does not require announcement</sub>
 - [ ] Added checklist items for additional operator tasks   <sub>or this PR does not require additional tasks</sub>
 
+## Author (hotfixes)
+
+- [ ] Added `h` tag to commit title and PR targets `prod`   <sub>or this PR does not include a temporary hotfix</sub>
+- [ ] Added `H` tag to commit title and PR targets `develop` <sub>or this PR does not include a permanent hotfix</sub>
```

Comment:
Unfortunately, this breaks the alignment of the sub tags. You'd have to shorten the text or move all the sub tags over.

Reply:
Added 7cabc77ed331cb46c7d61c51ff11aabf23b485b4 to align all the sub tags to https://github.com/DataBiosphere/azul/blob/3f4863032a586cfbdbd296fb101e6237ef638fd3/.github/pull_request_template.md?plain=1#L6 

Added fixup to move the hotfix section to match.